src/postprocess/dos.jl

```diff
@@ -68,65 +68,170 @@ function compute_ldos(scfres::NamedTuple; ε=scfres.εF, kwargs...)
 end
 
 """
-Projected density of states at energy ε for an atom with given i and l.
+Compute the projected density of states (PDOS) for all atoms and orbitals.
+
+    Input: 
+     -> εs               : Vector of energies at which the PDOS will be computed
+    Output:
+     -> pdos             : 3D array of PDOS, pdos[iε_idx, iproj, σ] = PDOS at energy εs[iε_idx] for projector iproj and spin σ
+     -> projector_labels : Vector of tuples (iatom, species, label, n, l, m) for each projector, that maps the iproj index to the corresponding atomic orbital. For details see the documentation for `atomic_orbital_projectors`.
```

Comment:
Formatting of the docs is a little unusual. Can you make it more similar to the other docstrings in the code ?